rayon-stable/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+extern crate rayon_core;
+
+pub mod par_iter;
+
+pub use rayon_core::prelude;
+
+pub use rayon_core::Configuration;
+pub use rayon_core::InitError;
+pub use rayon_core::dump_stats;
```

Comment:
this method does not need to be public, I think, and perhaps should be removed =)